projects/project-2/readings/Project 2 - SPARQL library

```diff
@@ -206,6 +254,10 @@ These annotations provide context and clarify the purpose of each element, which
 
 13. Detecting Properties Shared Across Multiple Classes (Level 3 - 20 points)
 
+Title: Shared Property Analysis
+Constraint Description: Identifies properties used by instances of multiple classes.
+Severity: Warning
+
```

Comment:
Not sure how discovering these relations between instances would be valuable for CCO, although it could be valuable for broader QC purposes

Reply:
Why wouldn't it be helpful to identify properties used by instances of multiple classes? Just curious.